src/content/en/agencies/css/directory.css

```diff
@@ -0,0 +1,502 @@
+/* Styles for Web Agencies Directory */
```

Comment:
This is pretty huge, is everything in here required?

Reply:
Yep, it's a lot of the same styles as other directories (https://developers.google.com/programs/launchpad/accelerator/directory/), plus styles for the new elements, like the lighthouse scores and agency cards.  Here is what it looks like on Devsite: https://devsite.googleplex.com/web/agencies/